### DISCUSSION TITLE
image distorted

### DISCUSSION THREAD

**Discussion by ronisegoly** [arkypita/LaserGRBL#1345] [General]
I used Photoshop to convert a profile image to sketch, and then tried to engrave it using lasergrbl
I attach the original sketch, the Gcode crated by the SW (quick save) and partial results.
I used 1BIT dithering and also line to line
The image is distorted at same point, but on SW screen it looks OK
Any clue why the result is distorted?
https://www.dropbox.com/s/xxjzbdubdfthrz6/IMG-20201211-WA0000.jpg?dl=0
https://www.dropbox.com/s/epr9mc9steg31hp/IMG-20201211-WA0000.nc?dl=0
![image](https://user-images.githubusercontent.com/5304613/108586870-a73cdf00-7359-11eb-8dc1-faf9d468fccf.png)

**Comment by arkypita** [comment]
http://lasergrbl.com/faq/ 
section: Inaccurate results and misalignments

**Comment by DevEvV** [comment]
Perhaps the Y-axis motor shaft is sliding in the pulley.

**Comment by ronisegoly** [comment]
Very strange behaviour
I engrave a lot with this machine so I'm quite used to it, and do not see
technical issues.
The issue I see is with some similar files, other not get distorted.
Also the distortion is always at the same point even if I crop the image,
and only if the engraving is done vertically.
I will keep watching





*Roni Segoly *

*+972-54-6466264*

* <http://en.wikipedia.org/wiki/Mohandas_Karamchand_Gandhi>*
